File: Sem1/Visual Studio/4- Abril/12-04/maior-ou-menor.py
num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))

if num1 > num2:
    print(f"O primeiro número {num1} é maior que o segundo número {num2}.")
elif num1 < num2:
    print(f"O segundo número {num2} é maior que o primeiro número {num1}.")
else:
    print("Os dois números são iguais.")

File: 1- Provas/SEM1/LOP/25-03/Prog/maior-ou-menor.c
#include <stdio.h>
#include<locale.h>

int main() {
	
	setlocale(LC_ALL,"");
	
    int num_numeros, i, maior, menor;
    int numero;
    
    printf("Quer saber qual numéro é maior e qual é menor?Vamos nessa!\n");
    printf("Qual é a quantidade de números que deseja comparar?\n");
    scanf("%d", &num_numeros);

    printf("Digite o %dº número: ", 1);
    scanf("%d", &maior);

    menor = maior;

    for(i = 2; i <= num_numeros; i++) {
        printf("Digite o %dº número: ", i);
        scanf("%d", &numero);

        if(numero > maior) {
            maior = numero;
        }

        if(numero < menor) {
            menor = numero;
        }
    }

    printf("O maior número informado foi: %d\n", maior);
    printf("O menor número informado foi: %d\n", menor);

    return 0;
}
